### PR TITLE
Drop from txpool tx that goes above the module line count limit

### PR DESCRIPTION
**Pull request by fab-10**

### CODE REVIEW

File: acceptance-tests/src/test/java/linea/plugin/acc/test/TransactionTraceLimitOverflowTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            GAS_PRICE.multiply(BigInteger.TEN));
+    final byte[] signedTxContractInteraction =
+        TransactionEncoder.signMessage(
+            txContractInteraction, Credentials.create(Accounts.GENESIS_ACCOUNT_TWO_PRIVATE_KEY));
+    final EthSendTransaction signedTxContractInteractionResp =
+        web3j.ethSendRawTransaction(Numeric.toHexString(signedTxContractInteraction)).send();
+
+    System.out.println(signedTxContractInteractionResp);
+
+    // this is the line count limit and should be selected
```
Comment (macfarla) [inline acceptance-tests/src/test/java/linea/plugin/acc/test/TransactionTraceLimitOverflowTest.java:78]:
```suggestion
    // this is under the line count limit and should be selected
```

File: acceptance-tests/src/test/java/linea/plugin/acc/test/TransactionTraceLimitOverflowTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            txData,
+            GAS_PRICE.multiply(BigInteger.TEN),
+            GAS_PRICE.multiply(BigInteger.TEN));
+    final byte[] signedTxContractInteraction =
+        TransactionEncoder.signMessage(
+            txContractInteraction, Credentials.create(Accounts.GENESIS_ACCOUNT_TWO_PRIVATE_KEY));
+    final EthSendTransaction signedTxContractInteractionResp =
+        web3j.ethSendRawTransaction(Numeric.toHexString(signedTxContractInteraction)).send();
+
+    System.out.println(signedTxContractInteractionResp);
```
Comment (macfarla) [inline acceptance-tests/src/test/java/linea/plugin/acc/test/TransactionTraceLimitOverflowTest.java:76]: System.out.println

File: acceptance-tests/src/test/java/linea/plugin/acc/test/TransactionTraceLimitOverflowTest.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  public void transactionOverModuleLineCountRemoved() throws Exception {
+    final SimpleStorage simpleStorage = deploySimpleStorage();
+    final Web3j web3j = minerNode.nodeRequests().eth();
+    final String contractAddress = simpleStorage.getContractAddress();
+    final Credentials credentials = Credentials.create(Accounts.GENESIS_ACCOUNT_ONE_PRIVATE_KEY);
+    final String txData = simpleStorage.add(BigInteger.valueOf(100)).encodeFunctionCall();
+
+    // this tx will not be selected since it goes above the line count limit
+    // but selection should go on and select the next one
+    final RawTransaction txContractInteraction =
```
Comment (macfarla) [inline acceptance-tests/src/test/java/linea/plugin/acc/test/TransactionTraceLimitOverflowTest.java:60]:
```suggestion
    final RawTransaction txModuleLineCountTooBig =
```

**Review by macfarla — APPROVED** [review]
LTM

File: acceptance-tests/src/test/java/linea/plugin/acc/test/TransactionTraceLimitOverflowTest.java (hunk 2 above)
Comment (fab-10) [reply acceptance-tests/src/test/java/linea/plugin/acc/test/TransactionTraceLimitOverflowTest.java:76]: removing

**Pull request merged**